photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Driver.scala

```diff
@@ -178,12 +178,12 @@ class Driver(val params: Params, val sparkContext: SparkContext, val logger: Pho
     logger.info(s"Number of scored items: $numScoredItems (s)\n")
     val scoredItemsToBeSaved =
       if (numOutputFilesForScores > 0 && numOutputFilesForScores != scoredItems.partitions.length) {
-        scoredItems.coalesce(numOutputFilesForScores)
+        scoredItems.repartition(numOutputFilesForScores)
```

Comment:
Why do we want repartition instead of coalesce in this case? Should coalesce in the training Driver also be changed to repartition?